pipelines/firebase/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""Loads data from firebase realtime database"""
+
+from typing import Any, Iterator, List, Sequence
+
+import dlt
+from dlt.extract.source import DltResource
+from dlt.common.typing import TDataItem
+
+from firebase_admin import credentials, db
+
+
+@dlt.source(name="firebase")
+def firebase_source(
+    project_id: str =  dlt.secrets.value,
```

Comment:
we have a building block for google service credentials, please see for example:
If your source/resource needs google credentials, just use dlt built-in credentials as we do in [google sheets](https://github.com/dlt-hub/pipelines/blob/master/pipelines/google_sheets/__init__.py#L26) and [google analytics](https://github.com/dlt-hub/pipelines/blob/master/pipelines/google_analytics/__init__.py#L32). Also note how credentials.to_native_credentials() is used to initialize google api client.